spec.go

```diff
@@ -9,7 +9,7 @@ import (
 	"io/fs"
 )
 
-//go:embed versions/1 versions/1/_dev versions/1/data_stream/_dev
+//go:embed versions/1 versions/1/integration/_dev versions/1/integration/data_stream/_dev
```

Comment:
@jsoriano Does Go ignore `_underscore-dirs` by design? I must have missed something here.

Reply:
Yes, there are certain parts of Go that ignore dirs prefixed with underscores and the `testdata` directory. For example I think that `go get` doesn't get these files by default, and `go test ./...` probably doesn't look for tests there.
I think that files referenced by `embed` are also seen by go get and other commands that would ignore them otherwise.

Reply:
Ok, I guess it's a workaround then. The PR is ready for a review round.